src/core/server/http/integration_tests/core_services.test.ts

```diff
@@ -266,9 +266,6 @@ describe('http service', () => {
 
       await kbnTestServer.request.get(root, '/new-platform/').expect(200);
 
-      // called twice by elasticsearch service in http route handler context provider
-      expect(clusterClientMock).toBeCalledTimes(2);
```

Comment:
This failed because the Saved Objects Client calls the clusterclient mock a third time when we create the route context. I could just increment this, but it feels like we're testing logic that isn't related to 'rewrites authorization header via authHeaders to make a request to Elasticsearch'. It feels like this test should focus on the behaviour of the headers not the amount of clusterclientmock calls.